src/Helpers/constants.js

```diff
@@ -572,6 +572,34 @@ export const SYSTEMS_ALLOWED_PARAMS = [
     'excluded'
 ];
 
+export const CVES_FILTER_PARAMS = [
+    'cvss_from',
```

Comment:
We also have `CVES_ALLOWED_PARAMS` let's reduce the duplicates by merging the arrays. 
somethine like that

```
[
...FILTERS,
'page', 
'page_size'
]
```
or 

```
[
...FILTERS,
...META
]
```